shoes-swt/lib/shoes/swt/common/fill.rb

```diff
@@ -25,10 +27,14 @@ def angle
           @angle || 0
         end
 
+        # Just clear it out and let next paint recreate and save our SWT color
+        def update_fill
+          @swt_fill = nil
```

Comment:
I guess if we added `cache` to the name the purpose of the variable would be clearer :)